src/Source/AbstractFileSource.php

```diff
@@ -77,10 +77,17 @@ public function getFileContent(bool $cache = false): string
 
             case EntityImportSourceContainer::RETRIEVAL_TYPE_HTTP:
                 $auth = [];
+                $options = [];
 
                 if (null !== $this->sourceModel->httpAuth) {
                     $httpAuth = \Contao\StringUtil::deserialize($this->sourceModel->httpAuth, true);
                     $auth = ['auth' => [$httpAuth['username'], $httpAuth['password']]];
+                    $options = $auth;
+                }
+
+                // Check if SSL verification should be done
+                if($this->sourceModel->dontCheckSSL) {
+                    $options['verify'] = false;
                 }
 
                 $event = $this->eventDispatcher->dispatch(BeforeAuthenticationEvent::NAME, new BeforeAuthenticationEvent($auth, $this->sourceModel));
```

Comment:
Could you please replace auth by options and fix the getter and setter; Afterwards please pass $event->getOptions() to storeValueToRemoteCache() and getContentFromUrl(). Then I guess we have it :-) Thanks for your support!

Reply:
Could you please rename the event to `ModifySourceHttpClientOptionsEvent` since this makes more sense. This way we can modify all options of guzzle and not only auth settings.